admin/load.php

```diff
@@ -336,3 +336,23 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Enable all non-experimental modules on plugin activation.
+ *
+ * @since 1.0.0
+ */
+function perflab_activation_hook() {
+	$modules          = perflab_get_modules();
+	$modules_settings = perflab_get_module_settings();
+
+	foreach ( $modules as $module_name => $module_data ) {
+		if ( ! $module_data['experimental'] ) {
+			$modules_settings[ $module_name ]['enabled'] = true;
```

Comment:
`$modules_settings[ $module_name ]` may not be set, we need to avoid potential PHP notices here. Since this should only run if the option is empty (see my other comment above), nothing will be set here, so we can do:
```suggestion
			$modules_settings[ $module_name ] = array( 'enabled' => true );
```